devicetypes/vision-raytseng/vision-4-in-1-motion-sensor.src/vision-4-in-1-motion-sensor.groovy

```diff
@@ -0,0 +1,552 @@
+/**
+ *  Vision 4-in-1 Motion Sensor
+ *
+ *  Author: Ray Tseng
+ */
+metadata {
```

Comment:
Please reformat the whole code and fix indentation (change 4 spaces to 1 tab) - applies to a whole file.